### PR TITLE
Removed create todo and Centered user avatar on project dashboard

### PR DESCRIPTION
**Pull request by ctanveer**
### Description

Removed create todo and centered user avatar on project dashboard

### Link to ticket

*Please add a link to the GitHub issue being addressed by this change.*

### Type

- [x] Fix
- [ ] Feature
- [ ] Cleanup 

### Screenshot of the result

*If your change affects the user interface, you should include a screenshot of the result with the pull request.*

### CODE REVIEW

File: app/Domain/Dashboard/Templates/show.blade.php [hunk 1]
```diff
@@ -94,11 +94,11 @@ class="{{ $isFavorite ? 'btn-active' : '' }}">
                 </x-global::content.card>
 
                 <x-global::content.card variation="content">
-                    <x-slot:card-context-buttons>
-                        <x-global::forms.button href="#/tickets/newTicket" content-role="tertiary">
+                    {{-- <x-slot:card-context-buttons>
```
Comment (marcelfolaron) [inline app/Domain/Dashboard/Templates/show.blade.php:97]: Please take out code that is not needed anhmore

File: app/Domain/Users/Templates/components/profile-image.blade.php [hunk 2]
```diff
@@ -4,4 +4,5 @@
 
 <img {{ $attributes->merge([
     'src' => BASE_URL . '/api/users?profileImage=' . $user['id'] .'&v='.format($user['modified'])->timestamp(),
+    'style' => "display: inline;"
```
Comment (marcelfolaron) [inline app/Domain/Users/Templates/components/profile-image.blade.php:7]: Does this actually work across the board? (Check project hub). Display inline will prevent us from doing some repositioning.

**Review by marcelfolaron — APPROVED** [review]

**Review by marcelfolaron — CHANGES_REQUESTED** [review]
Please resolve comments

File: app/Domain/Dashboard/Templates/show.blade.php (hunk 1 above)
Comment (ctanveer) [reply app/Domain/Dashboard/Templates/show.blade.php]: done

File: app/Domain/Users/Templates/components/profile-image.blade.php (hunk 2 above)
Comment (ctanveer) [reply app/Domain/Users/Templates/components/profile-image.blade.php]: removed inline style. Changed style.default.css

**Review by marcelfolaron — APPROVED** [review]

**Pull request merged**